fractal_server/app/schemas/_validators.py

```diff
@@ -44,14 +44,19 @@ def val(integer: int):
     return val
 
 
-def val_absolute_path(attribute: str):
+def val_absolute_path(attribute: str, accept_none: bool = False):
```

Comment:
We are adding the `accept_none`  here in `val_absolute_path`, but when we use `val_absolute_path` we never set `accept_none=True`.

Maybe this is a leftover from previous versions in this PR?

Reply:
I removed this argument.